assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+
+This document:
+
+- Measures existing documentation quality against the CNCF’s standards
+- Recommends specific and general improvements
+- Provides examples of great documentation as reference 
+- Identifies key improvements with the largest return on investment
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+## Project documentation 
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required) |
+
+### V1
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           |     |     | ✅  |     |     |
+| Content maintainability    |     | ✅  |     |     |     |
+| Content creation processes |     |     | ✅  |     |     |
+
+### V2
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           | ✅  |     |     |     |     |
+| Content maintainability    | ✅  |     |     |     |     |
+| Content creation processes | ✅  |     |     |     |     |
+
+### Comments
+
+There are two versions of notary that need to be supported with documentation.
+- V1 is housed with [The Update Framework](https://github.com/theupdateframework/notary), and the docs are included as a set of markdown files in the [/docs folder](https://github.com/theupdateframework/notary/tree/master/docs)
+- V2 is housed with The Notary Project (https://github.com/notaryproject/notaryproject) 
+
+**Information Architecture**: 
+- As there is no website as yet, and development is still ongoing, V2's Project Documentation is all contained within markdown files in the notaryproject repo.
+
+**New user content**:
+- What is there does a good job of explaining the project, but doesn't provide many docs focused on new users.
+
+**Content maintainability**:
+- The docs currently aren't versioned or localized, and is depending on GitHub for search.
+
+**Content creation processes**:
+- There are links to community discussions, but as there is no website yet there is no space to point to regarding docs creation.
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The notaryproject.dev site should primarily focus on the V2 work, but V1 needs to be represented as well.
```

Comment:
Should it? The v2 repo is, as discussed above, mostly spec work and internal documentation. I would make the recommendation that before _any_ site work starts, the project team meet with one of CNCF's technical writers/dev advocates to discuss exactly what needs to be documented for end users. I get the sense that they haven't thought about this from a user-centered perspective at all and that there are major gaps in the documentation.

Reply:
I agree that some more discussion with the maintainers is in order to try to sort out what needs to be documented for the end users -- currently it's hard to see how to start with the V2 project.

I think instead of saying that the notaryproject.dev site should focus on the V2 docs, it may be better to say that _initially_ the site should focus on V2 as a part of an MVP effort. [During my discussion with Steve](https://hackmd.io/2FmXhqxpTFGqM0yEjkmhcw?view), he mentioned that both versions should be represented. So there may be more to discuss regarding what is a priority for building out the site.

Reply:
I'm trying to keep this assessment and the website separate as they are two projects. But given they are so tightly linked, could the recommendations section include a prioritization of what needs to be done, or should that still only considered in the website project, keeping this assessment purely focused on the documentation tasks that should be done (regardless of when/how)?